x-pack/platform/plugins/shared/stack_connectors/common/openai/schema.ts

```diff
@@ -3,6 +3,8 @@
  * or more contributor license agreements. Licensed under the Elastic License
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
+ *
+ * PKI Funcitonality added by Antonio Piazza @antman1p
```

Comment:
I'm not aware that we do attributions like this - we can always track who made what changes via git blame, and it's possible this may mess up future "change all the copyright statements" changes we occaisonally make :-)